[WAF] Update Terraform instructions in migration guide (#7815)
cloudflare/cloudflare-docs: content/waf/reference/migration-guides/firewall-rules-to-custom-rules.md

Refer to the documentation about Terraform for [examples of configuring WAF custom rules using Terraform](/terraform/additional-configurations/waf-custom-rules/).

> ### Replacing your configuration using `cf-terraforming`

Reviewer: ```suggestion
### Replace your configuration using `cf-terraforming`
```
Changed only because I thought we were not writing the gerund anymore in titles.

Author: I guess it depends... We're not using it when providing a series of steps, which is the case. Thanks :)

> Consider using the [`cf-terraforming`](https://github.com/cloudflare/cf-terraforming) tool to generate the Terraform configuration for your current WAF custom rules (converted by Cloudflare from your firewall rules). Then, you can import the new resources to Terraform state.

Reviewer: ```suggestion
Use the [`cf-terraforming`](https://github.com/cloudflare/cf-terraforming) tool to generate the Terraform configuration for your current WAF custom rules (converted by Cloudflare from your firewall rules). Then, import the new resources to Terraform state.
```

Reviewer: Suggestion but you know your context better

Author: ```suggestion
You can use the [`cf-terraforming`](https://github.com/cloudflare/cf-terraforming) tool to generate the Terraform configuration for your current WAF custom rules (converted by Cloudflare from your firewall rules). Then, import the new resources to Terraform state.
```

Author: Adjusting your suggestion slightly (it's not mandatory to use the tool).

> The recommended steps for replacing your firewall rules (and filters) configuration in Terraform with a new ruleset configuration are the following:

Reviewer: ```suggestion
The recommended steps for replacing your firewall rules (and filters) configuration in Terraform with a new ruleset configuration are the following.
```

There were two of these in this sentence and the next one, so deleting one.
